test/unit/helpers/mocks/Util.js

```diff
@@ -311,6 +316,10 @@ fn.mockSetTimeout = function() {
   }
 };
 
+fn.hasTimeouts = function() {
+  return !!timeouts.length;
+};
```

Comment:
Is this used?

Reply:
it was used while I was hunting down these async issues. Most of those issues were resolved by patching the `render` in BaseController

Reply:
Great - let's remove the unused function then, since it is now unused.